core/templates/dev/head/pages/exploration_editor/ExplorationSaveService.js

```diff
@@ -93,6 +93,9 @@ oppia.factory('ExplorationSaveService', [
             };
             $scope.explorationId = (
               ExplorationContextService.getExplorationId());
+            $scope.explorationLink = !GLOBALS.DEV_MODE ? 'https://www.oppia.org/explore/' +
```

Comment:
I wonder, do you think the starting part of the link to the exploration should be made a defined constant? Same goes for the exploration link on the local host. It seems weird to be explicitly defining the links here.

Reply:
I thought that those parts of the link will remain fixed so it would be okay to.
Do we need to fetch them separately?

Reply:
@Aashish683 I guess thats true too, I don't see any reason it would change. I was just suggesting good programming practice in general. Its your call over here.

Reply:
Thanks !

Reply:
@Aashish683. Regarding this comment, I would revert back to my first comment. Explicitly defining them leads to fragile code. You could do something better like this:
```
$window.location.protocol + '//' + $window.location.host
```
This is how its done in components/embed_modal/ExplorationEmbedButtonService.js

Reply:
Done.